app/src/test/java/com/example/parkingsystemjava/mvp/presenter/DialogFragmentPresenterTest.java

```diff
@@ -0,0 +1,55 @@
+package com.example.parkingsystemjava.mvp.presenter;
+
+import com.example.parkingsystemjava.listener.ParkingDialogFragmentListener;
+import com.example.parkingsystemjava.presenter.contract.DialogFragmentContract;
+import com.example.parkingsystemjava.presenter.presenter.DialogFragmentPresenter;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.MockitoJUnitRunner;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.StrictStubs.class)
+public class DialogFragmentPresenterTest {
+    private DialogFragmentContract.PresenterContract presenter;
+    @Mock
+    private DialogFragmentContract.ViewContract view;
+    @Mock
+    private ParkingDialogFragmentListener dialogFragmentListener;
+    private final static String PARKING_LOT_STRING = "10";
+    private final static String PARKING_LOT_EMPTY_STRING = "";
+
+    @Before
+    public void setup() {
+        MockitoAnnotations.openMocks(this);
+        presenter = new DialogFragmentPresenter(view);
+    }
+
+    @Test
+    public void onSaveButtonPressedTest() {//el toast lo tengo en el view de la actividad ?? - no lo puedo llamar aca
```

Comment:
remove this comment pls and if it is necessary just put it in a new line and in english

Reply:
Done